app/specs/controllers/StudentsShowCtrl.spec.js

```diff
@@ -0,0 +1,48 @@
+// disable max-nested-callbacks linter for this test
+// it is good to have a good separation of contexts inside tests
+
+/* eslint-disable max-nested-callbacks */
+
+'use strict';
+
+define(['paw',
+'angular-mocks',
+'controllers/StudentsShowCtrl'],
+function() {
+  describe('Students Show Ctrl', function() {
+    beforeEach(module('paw'));
+
+    var $controller, controller, expectedStudent;
+
+    beforeEach(inject(
+      function(_$controller_) {
+        $controller = _$controller_;
+        controller = $controller('StudentsShowCtrl');
+        expectedStudent =
+        {
+          docket: '55019',
+          firstName: 'Matías',
+          lastName: 'Mercado',
+          email: 'mmercado@itba.edu.ar',
+          genre: 'Masculino',
+          dni: '38917403',
+          birthday: '1995-05-04',
+          address: {
+            country: 'Argentina',
+            city: 'Buenos Aires',
+            neighborhood: 'Almagro',
+            number: '682',
+            street: 'Corrientes',
+            floor: '2',
+            door: 'A',
+            telephone: '1544683390',
+            zipCode: '1100'
+          }
+        };
+      }));
+
+      it('loads the required student', function() {
+        expect(controller.student).toEqual(expectedStudent);
+      });
```

Comment:
Test case missing: you should be testing that the controller's docket is being loaded from the $routeParams service. Take a look at the `StudentsIndexCtrl.spec.js` from #63 

However, if you've changed the docket assignment from `this.docket` to `var docket` as suggested above, you won't be able to test the controllers variable (as it is local to that function). What you would have to do is to use a [Jasmine Spy](https://jasmine.github.io/edge/introduction.html#section-Spies) and check that the $routeParam.docket is being requested. You can also check other spec files for further help. If you cannot manage it, just tell me.